IOPool/Input/src/PoolSource.cc

```diff
@@ -319,6 +365,8 @@ namespace edm {
         ->setComment("Size of ROOT TTree TBasket cache. Affects performance.");
     desc.addUntracked<unsigned int>("setRunNumber", 0U)
         ->setComment("If non-zero, change number of first run to this number. Apply same offset to all runs. Allowed only for simulation.");
+    desc.addUntracked<std::vector<unsigned int> >("setRunNumberForEachLumi", std::vector<unsigned int>())
```

Comment:
There already is such protection, but I will talk with David to see if this protection can be done in the descriptions instead.